weldx/asdf/file.py

```diff
@@ -59,9 +70,15 @@ def reset_file_position(fh: SupportsFileReadWrite):
 DEFAULT_ARRAY_COPYING = True
 """Stored Arrays will be copied to memory, or not. If False, use memory mapping."""
 
+_PROTECTED_KEYS = (
+    "history",
+    "asdf_library",
+)
+"""These keys are not not seen, nor can they be manipulated."""
```

Comment:
> nor can they be manipulated

Would you bet on this? 😄

Reply:
Not if you suggest to make a bet :) What have you figured?

Reply:
Nothing in particular, just that in Python nothing is really protected from being manipulated. It was just a fun- comment. 😉

Reply:
True, you can also re-define integers (try swapping one for zero is total fun).